addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -22,15 +22,11 @@
 class AccountMove(models.Model):
     _inherit = 'account.move'
 
-    l10n_it_edi_transaction = fields.Char(copy=False, string="FatturaPA Transaction")
-    l10n_it_edi_attachment_id = fields.Many2one('ir.attachment', copy=False, string="FatturaPA Attachment", ondelete="restrict")
-
-    l10n_it_stamp_duty = fields.Float(default=0, string="Dati Bollo", readonly=True, states={'draft': [('readonly', False)]})
-
+    l10n_it_edi_transaction = fields.Char(copy=False, string="It Edi Transaction")
+    l10n_it_edi_attachment_id = fields.Many2one('ir.attachment', copy=False, string="It Edi Attachment", ondelete="restrict")
+    l10n_it_stamp_duty = fields.Float(default=0, string="Stamp Duty", readonly=True, states={'draft': [('readonly', False)]})
     l10n_it_ddt_id = fields.Many2one('l10n_it.ddt', string='DDT', readonly=True, states={'draft': [('readonly', False)]}, copy=False)
-
     l10n_it_einvoice_name = fields.Char(compute='_compute_l10n_it_einvoice')
-
```

Comment:
Removed unnecessary spaces, translated Italian terms in the code (and added to the PO file)

Reply:
We can discuss "It Edi Transaction" and "It Edi Transaction"... maybe you like "Edi Transaction (Italy)" or something